app/views/guidance/performance_profiles_outside.html.erb

```diff
@@ -0,0 +1,32 @@
+<%= render PageTitle::View.new(text: "Sign off your list of trainees from #{@previous_academic_cycle_label} for the performance profiles publication") %>
+
+<%= content_for(:breadcrumbs) do %>
+  <%= render GovukComponent::BackLinkComponent.new(
+    text: "How to use this service",
+    href: guidance_path,
+    ) %>
+<% end %>
+
+<div class="govuk-grid-row">
+   <div class="govuk-grid-column-two-thirds-from-desktop">
+      <h1 class="govuk-heading-l" id="signing-off-your-list-of-trainees-for-the-performance-profiles-publication">Signing off your list of trainees for the performance profiles publication</h1>
+      <p class="govuk-body">Each year you need to sign off your list of trainee teachers from the previous academic year. You’ll be told by email when you need to do this.</p>
+      <p class="govuk-body">The data you sign off will be included in the annual initial teacher training performance profiles publication. This will be published on GOV.UK.</p>
+      <h2 class="govuk-heading-m" id="finding-your-trainee-data">Finding your trainee data</h2>
+      <p class="govuk-body">You’ll need a list of all registered trainees who studied in the previous academic year. A report will be available showing the data you need to check.</p>
+      <h2 class="govuk-heading-m" id="checking-your-trainee-data">Checking your trainee data</h2>
+      <p class="govuk-body">You’ll need to check that all your trainees for the previous academic year are listed. For each trainee you’ll need to check:</p>
+      <ul class="govuk-list govuk-list--bullet">
+         <li>course details</li>
+         <li>status</li>
+         <li>the date they were awarded qualified teacher status (QTS) or early years teacher status (EYTS), or deferred or withdrew from the course - unless they’re still studying</li>
+      </ul>
+      <h2 class="govuk-heading-m" id="fixing-mistakes-in-your-trainee-data">Fixing mistakes in your trainee data</h2>
+      <p class="govuk-body">You’ll need to fix any mistakes before you sign off the data. How you’ll do this depends on whether you’re a provider of school centred initial teacher training (SCITT) or a higher education institution (HEI).</p>
+      <h2 class="govuk-heading-m" id="signing-off-your-trainee-data">Signing off your trainee data</h2>
+      <p class="govuk-body">A senior person from your organisation will need to complete a form to sign off the data. This should not be the same person who entered the data.</p>
+      <h2 class="govuk-heading-m" id="how-your-trainee-data-will-be-used">How your trainee data will be used</h2>
+      <p class="govuk-body">Read the ITT performance profiles methodology (opens in a new tab) to find out how your trainee data will be used.</p>
+      <p class="govuk-body">Some trainees will not be included in the ITT performance profiles publication. For example, trainees who withdrew within 90 days of the course start will not be included.</p>
+   </div>
+</div>
```

Comment:
There's a few lines missing new line like here. I wonder why the linter doesn't capture these